### ISSUE TITLE
Cannot edit the indexed directories

### ISSUE THREAD

**Issue by petko10** [vanilla-music/vanilla#1024]
- Vanilla Music Version: 1.0.85
- Android Version: 9
- Phone vendor / model: Xiaomi Mi A1
- Application was downloaded from (use `[x]`):
  - [ x] Play Store
  - [ ] F-Droid
  - [ ] Other (Nightly Build)
- Did this work in a previous version?
  - [x] Yes
  - [ ] No
  - [ ] NA / Don't know.

## Description
Cannot add new folders to the indexed directories
....

## Steps to reproduce the issue
Install Vanilla (I cleared data/cache to check) and try to press Edit in the Media library settings menu. Add a folder and it will not appear in the list.

**Comment by cathos** [comment]
I am having the same issue, with version 1.0.85 from F-Droid on Android 10 on an Essential Phone PH-1.

**Comment by adrian-bl** [comment]
How do you try to 'add' the folder?

You realize that you need to 'long press' on the folder to select whether or not to include or exclude it?

**Comment by cathos** [comment]
I tried to go into the music directory from Settings>Indexed Directories>Edit and then hit save. Long pressing to add a directory was not at all obvious. I tried that just now, and it seemed to work. My music directory now shows up under Indexed Directories.

However, is there something you can change to make that more obvious? Or allowing to add the current directory by hitting Save while in the directory with nothing selected?

**Comment by petko10** [comment]
Agreed, I saw just now that there's a notification that pops up when you open the prompt. I'd suggest to make it a fixed text label on the top/bottom, because it can be very easily overlooked.

**Comment by adrian-bl** [comment]
Yeah, the UI there isn't that great:

I hacked something up in https://github.com/vanilla-music/vanilla/commit/621c917bf4b9dcefcd55fe61d34d58404a94582a which adds a dropdown/spinner to the bottom of the view to select the state of the current directory.

**Comment by atrull** [comment]
This is a really naff problem. Got to be a better way to show this functionality.
